features/storage/csi.feature

```diff
@@ -37,3 +37,16 @@ Feature: CSI testing related feature
       | /mnt/cinder/hello |
     Then the step should succeed
     And the output should contain "Hello OpenShift Storage"
+
+  # @author chaoyang@redhat.com
+  # @case_id OCP-30787
+  @admin
+  Scenario: CSI images checking in stage env
+    Given I use the "csihostpath" project 
+    And pod named "my-csi-app" becomes ready
```

Comment:
@chao007 this step doesn't seem to exist...  I think it should be https://github.com/openshift/verification-tests/blob/master/features/step_definitions/pod.rb#L28.   Please make sure the test pass in jenkins (or at least locally) first before submitting a PR